Assets/MRTK/SDK/StandardAssets/Controllers/DebugControllers/ControllerRight.prefab

```diff
@@ -28,7 +28,7 @@ Transform:
   m_LocalPosition: {x: 0, y: 0, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children:
-  - {fileID: 105240301390012451}
+  - {fileID: 4711765180558349674}
```

Comment:
@RogPodge here are the serialization changes I needed to get these prefabs to work

Reply:
Looks good to me!